### ISSUE TITLE
Motion Representation during training

### ISSUE THREAD

**Issue by Evm7** [neu-vi/MARDM#2]
Hi there. First of all, thanks a lot for your great insights on motion representation and its impact on  AE-like approaches as well as diffusion; it is greatly appreciated and the results seem promising.

I wanted to validate the performance of the model but observed that AE checkpoints are not provided (as mentioned in another issue). Then, I tried training the AE but observed issues on the train_AE.py code.

For instance, while the data loaded is directly the new_joint_vecs (with 263-dim, w/ redundant information), the model takes only 67-dim (as mentioned in the paper). Thus there is an error in shapes during the AE forward pass. I assume there is missing a preprocessing of features to 3D joints (3*22 joints=66). 

Similarly, I am wondering what is with the additional 67 dimension of the data, as I could not find the information about this in the paper (i.e., Njoints*3 = 66, what is the +1 to make the 67-dim).

Thanks

**Comment by cr8br0ze** [comment]
Hi there, thank you for your interest in our work. I totally forgot to provide the pre-trained AE model and the README.md file has been updated with the downloading link.
For Training the AE from scratch, I believe it's because the original motion data (263 dim) is only processed to (67 dim) with the updated mean and std in utils/datasets file (specifically in current version line 57). So if you can recalculate the mean and std following data preparation instructions or use our obtained ones (change the commenting of mean and std in train or eval files), the problem should be solved.
In paper we wrote joints number *3 +1, because it's better to write this than 1+2+1+(joints number-1) *3 => (root_rot_velocity (B, seq_len, 1), root_linear_velocity (B, seq_len, 2), root_y (B, seq_len, 1), ric_data (B, seq_len, (joint_num - 1)*3)). It's still the first four essential motion components as mentioned in paper.
